packages/itwin/reports-config-widget/src/widget/components/BulkExtractor.ts

```diff
@@ -95,7 +143,7 @@ export default class BulkExtractor {
     return ExtractionStates.Failed;
   }
 
-  private async getSingleState(runId: string, accessToken: string): Promise<ExtractorState> {
+  private async getState(runId: string, accessToken: string): Promise<ExtractorState> {
```

Comment:
You can take `accessToken` with `await this._accessToken()`.

Reply:
`AccessToken` is now awaited inside the method